lib/dfe/analytics.rb

```diff
@@ -96,19 +92,21 @@ def self.configure
     end
 
     def self.initialize!
-      unless defined?(ActiveRecord)
-        # bail if we don't have AR at all
-        Rails.logger.error('ActiveRecord not loaded; DfE Analytics not initialized')
-        return
-      end
-
       unless Rails.env.production? || File.exist?(Rails.root.join('config/initializers/dfe_analytics.rb'))
         message = "Warning: DfE Analytics is not set up. Run: 'bundle exec rails generate dfe:analytics:install'"
         Rails.logger.error(message)
         puts message
         return
       end
 
+      if defined?(ActiveRecord)
+        setup_entities
+      else
+        Rails.logger.info('ActiveRecord not loaded; DfE Analytics will only track web requests.')
```

Comment:
I think the message should say:
`DfE Analytics will only track non database events`
as we have custom events as well as web request events in some services.